keycloak/keycloak.go

```diff
@@ -100,6 +100,40 @@ func (k *Keycloak) GetUser(ctx context.Context, userID string) (*User, error) {
 	}
 
 	user := &User{
+		ID:                   gocloak.PString(kcuser.ID),
```

Comment:
Not necessary in this PR, but it would be nice to move the `gocloak.User` -> `User` conversion logic into a helper function to avoid duplication.